src/components/PaymentRequestActionMenu/PaymentRequestActionMenu.stories.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { ComponentStory, ComponentMeta } from '@storybook/react';
+
+import PaymentRequestActionMenu from './PaymentRequestActionMenu';
+
+export default {
+  title: 'Payment Request Row Action Menu',
+  component: PaymentRequestActionMenu,
+} as ComponentMeta<typeof PaymentRequestActionMenu>;
+
+const Template: ComponentStory<typeof PaymentRequestActionMenu> = (args) => <PaymentRequestActionMenu {...args} />;
+
+export const ActionMenuTemplate = Template.bind({});
+ActionMenuTemplate.args = {
+  onDuplicate: () => {
+    alert('You clicked Duplicate!');
```

Comment:
You could replace this with action('You clicked Duplicate!') and it will show in the Actions pane